fearless/Modules/AccountCreate/AccountCreateViewLayout.swift

```diff
@@ -0,0 +1,343 @@
+        let view = UIFactory().createExpandableActionControl()
+        view.backgroundColor = .black
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    private let detailsLabel: UILabel = {
+        let label = UILabel()
+        label.font = .p1Paragraph
+        label.textColor = R.color.colorLightGray()
+        label.textAlignment = .left
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        return label
+    }()
+
+    private let detailsLabelView: UIView = {
+        let view = UIView()
+        view.backgroundColor = R.color.colorBlack()!
+        return view
+    }()
+
+    private let substrateDerivationContainerView: TriangularedView = {
+        let view = TriangularedView()
+        view.shadowOpacity = Constants.shadowOpacity
+        view.fillColor = UIColor.clear
+        view.highlightedFillColor = UIColor.clear
+        view.strokeColor = R.color.colorGray()!
+        view.strokeWidth = Constants.strokeWidth
+        return view
+    }()
+
+    private let ethereumDerivationContainerView: TriangularedView = {
+        let view = TriangularedView()
+        view.shadowOpacity = Constants.shadowOpacity
+        view.fillColor = UIColor.clear
+        view.highlightedFillColor = UIColor.clear
+        view.strokeColor = R.color.colorGray()!
+        view.strokeWidth = Constants.strokeWidth
+        return view
+    }()
+
+    private let advancedContainerView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = UIConstants.bigOffset
+        return stackView
+    }()
+
+    private let advancedAppearanceAnimator = TransitionAnimator(
+        type: .push,
+        duration: 0.35,
+        subtype: .fromBottom,
+        curve: .easeOut
+    )
+
+    private let advancedDismissalAnimator = TransitionAnimator(
+        type: .push,
+        duration: 0.35,
+        subtype: .fromTop,
+        curve: .easeIn
+    )
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupLayout()
+        configure()
+    }
+
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension AccountCreateViewLayout {
+    func handleKeyboard(frame: CGRect) {
+        nextButton.snp.updateConstraints { make in
+            make.bottom.equalTo(safeAreaLayoutGuide).inset(frame.height + UIConstants.bigOffset)
+        }
+    }
+}
+
+private extension AccountCreateViewLayout {
+    private func configure() {
+        contentView.stackView.arrangedSubviews.forEach { $0.backgroundColor = R.color.colorBlack() }
+        advancedContainerView.isHidden = !expandableControl.isActivated
+        expandableControl.addTarget(self, action: #selector(actionExpand), for: .touchUpInside)
+    }
+
+    @objc
+    private func actionExpand() {
```

Comment:
I think it's better to use *ViewLayout class as layout and don't store any logic there. This piece of code should be replaced to ViewController class. agree?